test/cql-pytest/test_uda.py

```diff
@@ -136,4 +136,20 @@ def test_reduce_function(scylla_only, cql, test_keyspace):
                 aggr_body = f"(int) SFUNC {row_f} STYPE bigint REDUCEFUNC {reduce_f} FINALFUNC {final_f} INITCOND 0"
                 with new_aggregate(cql, test_keyspace, aggr_body) as aggr_f:
                     result = cql.execute(f"SELECT aggregate_name, reduce_func, state_type FROM system_schema.scylla_aggregates WHERE keyspace_name = '{test_keyspace}' AND aggregate_name = '{aggr_f}' AND argument_types = ['int']").one()
-                    assert result == (aggr_f, reduce_f, 'bigint')
\ No newline at end of file
+                    assert result == (aggr_f, reduce_f, 'bigint')
+
+def test_collection_initcond(scylla_only, cql, test_keyspace):
+    schema = "id int primary key"
+    row_body = "(acc list<int>, val int) RETURNS NULL ON NULL INPUT RETURNS list<int> LANGUAGE lua AS 'table.insert(acc, val); return acc'"
+    final_body = "(acc list<int>) RETURNS NULL ON NULL INPUT RETURNS list<int> LANGUAGE lua AS 'table.sort(acc); return acc'"
```

Comment:
Let's add a test case for a nested structure too, just in case - like, a list of sets of udts.